hometask1/hometask1_1.js

```diff
@@ -0,0 +1,6 @@
+var stdin = process.openStdin();
```

Comment:
Consider using "const" and "let" instead of "var".
Here, stdid variable doesn't change in the current scope - therefore "const" would be ideal.